SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -193,7 +198,7 @@ var splunkRumInitializeCalledTime = Date()
      
      */
     @discardableResult
-    @objc public class func initialize(beaconUrl: String, rumAuth: String, options: SplunkRumOptions? = nil) -> Bool {
+    @objc internal class func initialize(beaconUrl: String, rumAuth: String, options: SplunkRumOptions? = nil) -> Bool {
```

Comment:
Since moving to the new pattern is a breaking change, perhaps we should mark this function along with the previous Options (if it's possible for non-funcs) deprecated? And only after a while remove it completely

Reply:
Yeah that makes sense.  I can revert this and mark this and the SplunkRumOptions as deprecated since the options are not going to be exposed in a future release.